frontend/packages/react-app/src/components/organisms/DownloadEmailCsvPane/index.tsx

```diff
@@ -0,0 +1,138 @@
+ */
+
+import * as React from "react";
+import { Paper, Typography, Box, Button } from "@material-ui/core";
+import { useState } from "react";
+import { EMAIL_ACTIONS, EmailState } from "../../../reducers/email";
+import { CSVLink } from "react-csv";
+import styled from "styled-components";
+
+export interface DownloadEmailCsvPaneProps {
+  readonly tokenAddress: string;
+  readonly campaignAddress: string;
+  readonly emailState: EmailState;
+  readonly emailDispatch: React.Dispatch<EMAIL_ACTIONS>;
+}
+
+const StyledCSVLink = styled(CSVLink)`
+  text-decoration: none;
+`;
+
+const DownloadEmailCsvPane: React.FC<DownloadEmailCsvPaneProps> = ({
+  tokenAddress,
+  campaignAddress,
+  emailState,
+  emailDispatch,
+}) => {
+  const [isDownloaded, setIsDownloaded] = useState(false);
+  const urlList = emailState.rawTargets.map(
+    (uuid) =>
+      `${window.location.origin}/explore/${tokenAddress}/distributors/${emailState.distributorAddress}/campaigns/${campaignAddress}?uuid=${uuid}`
+  );
+  const emailUrlPair = emailState.emailList.map((email, index) => {
+    return [email, urlList[index]];
+  });
+  const csvData = [["Email", "UUID"], ...emailUrlPair];
+
+  return (
+    <Box mt={2}>
+      <Paper>
+        <Box p={4}>
+          <Box m={"auto"} width={[3 / 4]} mt={2}>
+            <Typography variant={"h4"}>4. Download CSV</Typography>
+            <Box mt={2}>
+              <Typography display="inline">
+                A campaign is created successfully for
+              </Typography>
+              <Typography display="inline" color="primary">
+                {" "}
+                {emailState.rawTargets.length} Email addresses
+              </Typography>
+              <Typography display="inline">
+                {" "}
+                which you have uploaded.
+              </Typography>
+              <Typography>
+                Download CSV contains Email and campaign URL pair for each
+                addresses and send a emails for them!
+              </Typography>
+              <Typography>
+                You can send a Email using batch email sending service such as{" "}
+                <a
+                  href="https://www.mergemail.co/"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  MergeMail
+                </a>
+                ,{" "}
+                <a
+                  href="https://www.gmass.co/"
+                  target="_blank"
+                  rel="noreferrer"
+                >
+                  GMass
+                </a>{" "}
+                or{" "}
+                <a href="https://yamm.com/" target="_blank" rel="noreferrer">
+                  YAMM
+                </a>
+                .
+              </Typography>
+              <Typography>
+                Please note, this file will be deleted after you move from this
+                page, so please be careful.
+              </Typography>
```

Comment:
Minor grammar corrections (from Grammarly)

A campaign is created successfully for 1 Email address that you have uploaded. Download CSV contains Email and campaign URL pair for each address and send an email for them! You can send an Email using a batch email sending service such as MergeMail, GMass, or YAMM. Please note, this file will be deleted after you move from this page, so please be careful.